component/src/main/wso2/kafka/consumer.bal

```diff
@@ -188,125 +189,140 @@ public type SimpleConsumer object {
-        P{{onPartitionsAssigned}} function will be executed if partitions are assigned this consumer
-    }
+    # Subscribes to consumer to external Kafka broker topic with rebalance listening is enabled.
+    #
+    # + topics - topic array to be subscribed.
+    # + onPartitionsRevoked - function will be executed if partitions are revoked from this consumer.
+    # + onPartitionsAssigned - function will be executed if partitions are assigned this consumer.
+    # + returns - Returns an error if encounters an error, returns nil otherwise.
     public extern function subscribeWithPartitionRebalance(string[] topics,
     function(ConsumerAction consumerActions, TopicPartition[] partitions) onPartitionsRevoked,
     function(ConsumerAction consumerActions, TopicPartition[] partitions) onPartitionsAssigned) returns error?;
 
-    documentation { Assign consumer to external Kafka broker set of topic partitions
-        P{{partitions}} topic partitions to be assigned
-    }
+    # Assign consumer to external Kafka broker set of topic partitions.
+    #
+    # + partitions - topic partitions to be assigned.
+    # + returns - Returns an error if encounters an error, returns nil otherwise.
     public extern function assign(TopicPartition[] partitions) returns error?;
 
-    documentation { Returns current offset position in which consumer is at
-        P{{partition}} topic partitions in which the position is required
-        R{{}} position in which the consumer is at in given Topic partition
-    }
+    # Returns current offset position in which consumer is at.
+    #
+    # + partition - topic partitions in which the position is required.
+    # + return - position in which the consumer is at in given Topic partition if executes successfully, error otherwise.
     public extern function getPositionOffset(TopicPartition partition) returns int|error;
 
-    documentation { Returns current assignment of partitions for a consumer
-        R{{}} Assigned partitions array for consumer
-    }
+    # Returns current assignment of partitions for a consumer.
+    #
+    # + return - Assigned partitions array for consumer if executes successfully, error otherwise.
     public extern function getAssignment() returns TopicPartition[]|error;
 
-    documentation { Returns current subscription of topics for a consumer
-        R{{}} subscribed topic array for consumer
-    }
+    # Returns current subscription of topics for a consumer.
+    #
+    # + return - subscribed topic array for consumer if executes successfully, error otherwise.
     public extern function getSubscription() returns string[]|error;
 
-    documentation { Returns current subscription of topics for a consumer
-        P{{partition}} partition in which offset is returned for consumer
-        R{{}} committed offset for consumer for given partition
-    }
+    # Returns current subscription of topics for a consumer.
```

Comment:
This returns committed offset of a consumer, for a given partition. The definition is duplicated from the above, not changed.

Reply:
Fixed in 037ee71